arm9/lib/include/MATH.h

```diff
@@ -0,0 +1,8 @@
+#ifndef GUARD_MATH_H
+#define GUARD_MATH_H
+
+#include "types.h"
+
```

Comment:
imo both crc and dgt should be included here, so that we can just #include "MATH.h" and have everything

Reply:
You were looking at the wrong file. Renamed to avoid confusion